src/NodaTime/CalendarSystem.cs

```diff
@@ -232,10 +236,31 @@ public static CalendarSystem ForId(string id)
         /// <returns>A Hebrew calendar system for the given month numbering.</returns>
         public static CalendarSystem GetHebrewCalendar(HebrewMonthNumbering monthNumbering)
         {
-            Preconditions.CheckArgumentRange(nameof(monthNumbering), (int) monthNumbering, 1, 2);
-            return HebrewCalendars.ByMonthNumbering[((int) monthNumbering) - 1];
+            Preconditions.CheckArgumentRange(nameof(monthNumbering), (int)monthNumbering, 1, 2);
```

Comment:
Our Visual Studio configs don't have the same settings for configuration. This is going to be a nightmare because I need to touch existing files, not just add new ones! Is there a way to share your VS settings?